src/frontend-pwa/src/Type/MultipleLocations.ts

```diff
@@ -0,0 +1,5 @@
+import SingleLocation from './SingleLocation';
+
+type MultipleLocations = SingleLocation[];
```

Comment:
Can we name this something with more meaning, example 'LocationsArray'.

Reply:
Sure can!